docs/products/postgresql/howto/monitoring-query-performance

```diff
@@ -0,0 +1,34 @@
+Monitoring Query Performance with Datadog
```

Comment:
Hi @harshini-rangaswamy, why is this file here?

Reply:
This shouldn't be in here! Thanks for spotting this one.